test/functional/testUtil.go

```diff
@@ -101,7 +101,7 @@ func GetBicepRecipeRegistry() string {
 func GetBicepRecipeVersion() string {
 	defaultVersion := os.Getenv("BICEP_RECIPE_TAG_VERSION")
 	if defaultVersion == "" {
-		defaultVersion = "latest"
+		defaultVersion = "pr-ce9d28f9ac"
```

Comment:
Were changes in this file intended or were meant for local testing?

Reply:
Thanks for catching this!